src/elin_bjoernson_ex/ex01/deck_by_comp.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+
+SUITS = ('C', 'S', 'H', 'D')
+VALUES = range(1, 14)
+
+
+def deck_loop():
+    deck = []
+    for suit in SUITS:
+        for val in VALUES:
+            deck.append((suit, val))
+    return deck
+
+
+def deck_comp():
+    deck = [(s, v) for s, v in [(s, v) for s in SUITS for v in VALUES]]
```

Comment:
The result looks nice, but this is too complicated.
`deck = [(s, v) for s in SUITS for v in VALUES]` is enough.